src/dunst.c

```diff
@@ -23,7 +23,7 @@
 #include "x11/x.h"
 
 #ifndef VERSION
-#define VERSION "version info needed"
+#error "no version info given"
```

Comment:
If we're going to fail in this case the entire `ifndef` block is redudant, the compiler will fail because of the undefined VERSION macro when it's used.